storybook/pages/BuyCryptoModalPage.qml

```diff
@@ -47,6 +49,22 @@ SplitView {
                 d.debounceFetchProviderUrl()
             }
         }
+
+        readonly property var currencyStore: CurrenciesStore {}
+        readonly property var assetsStore: WalletAssetsStore {
+            id: thisWalletAssetStore
+            walletTokensStore: TokensStore {
+                plainTokensBySymbolModel: TokensBySymbolModel {}
+            }
+            readonly property var baseGroupedAccountAssetModel: GroupedAccountsAssetsModel {}
+            assetsWithFilteredBalances: thisWalletAssetStore.groupedAccountsAssetsModel
+        }
+        readonly property BuyCryptoParamsForm buyCryptoInputParamsForm: BuyCryptoParamsForm{
+            selectedWalletAddress: "0x7F47C2e18a4BBf5487E6fb082eC2D9Ab0E6d7240"
+            selectedNetworkChainId: 11155111
+            selectedTokenKey: "ETH"
+        }
```

Comment:
This is basically bad practice that we should avoid. The transformations are duplicated via copy-paste in real store and stub and we try to mimic the setup from the app here in page. Hopefully it will all simplify after doing the second part of alignments.